patches/tModLoader/Terraria.ModLoader/ModInternals.cs

```diff
@@ -69,7 +69,10 @@ partial class Mod
 							Language.GetTextValue("tModLoader.LoadErrorTextureFailedToLoad", path), t.Exception);
 
 					var tex = t.Result;
+					
 					tex.Name = Name + "/" + path;
+					tex.Tag = Name;
```

Comment:
Tag seems redundant here, why can't we just use `Name`? I'd like to leave `Tag` for other modders to use